src/items/traits.md

```diff
@@ -62,12 +62,12 @@ trait Seq<T> {
 }
 ```
 
-## Object Safety
+## Trait-Object Safety <a id="object-safety"></a>
 
-Object safe traits can be the base trait of a [trait object]. A trait is
-*object safe* if it has the following qualities (defined in [RFC 255]):
+An object-safe trait can be used in a `dyn Trait` type, and be the base trait of a [trait object].
+A trait is *object safe* if it has the following qualities (defined in [RFC 255]):
```

Comment:
```suggestion
A dyn-compatible trait can be used in a trait object type (`dyn Trait`), and be the base trait of a [trait object].
A trait is *dyn compatible* if it has the following qualities (defined in [RFC 255]):
```

Reply:
Unrelated to the renaming: The dyn-compatibility rules have been updated over time (for example, "dyn-incompatible" associated *types* marked with `where Self: Sized`  no longer render the trait dyn-incompatible). So stating that the rules were defined in RFC 255 is misleading or at the very least outdated. That should probably be changed as part of a separate PR though.